packagehandlers/nugetpackagehandler.go

```diff
@@ -36,8 +34,8 @@ func (nph *NugetPackageHandler) UpdateDependency(vulnDetails *utils.Vulnerabilit
 }
 
 func (nph *NugetPackageHandler) updateDirectDependency(vulnDetails *utils.VulnerabilityDetails) (err error) {
-	var assetsFilePaths []string
-	assetsFilePaths, err = getAssetsFilesPaths()
+	var descriptorFilesFullPaths []string
+	descriptorFilesFullPaths, err = nph.GetAllDescriptorFilesFullPaths([]string{dotnetAssetsFilesSuffix})
```

Comment:
The GetAllDescriptorFilesFullPaths function receives a variadic operator. This allows you to send either no string, a single string, or a list of strings as arguments to the function call.

```suggestion
	descriptorFilesFullPaths, err = nph.GetAllDescriptorFilesFullPaths(dotnetAssetsFilesSuffix)
```

Reply:
we cannot have 2 variadic arguments in a Go function, therefore one of the arguments must be provided in a different way. Since the suffixes are "mandatory" but the exclude patterns are not, I chose to use the variadic variable for the exclude patterns